Multisig/Logic/Models/Transaction.swift

```diff
@@ -79,7 +79,8 @@ extension Transaction {
                     data: data?.data,
                     operation: operation)
 
-            self.nonce = UInt256String(estimationResult.latestNonce.value + 1)
+            self.nonce = UInt256String(
+                estimationResult.latestNonce.value == 0 ? 0 : estimationResult.latestNonce.value + 1)
```

Comment:
I would like to check if there is "recommendedNonce" first, otherwise we do this logic

Reply:
As I understand it, `recommendedNonce` will be a part of the new endpoint.